src/Altinn.App.Core/Infrastructure/Clients/Storage/DataClient.cs

```diff
@@ -70,33 +77,42 @@ public async Task<DataElement> InsertFormData<T>(
         int instanceOwnerPartyId,
         string dataType
     )
+        where T : notnull
```

Comment:
Need to notify these guys? https://altinn.studio/repos/brg/starteogdrive/src/commit/e3b62d445a112e2db65004b90c1f0575ee3c52ab/App/Services/BRGProfileService.cs#L618-L631

Reply:
I'm not sure. It's only a warning, and they should check for null because I'm pretty sure the app instance is broken beyond repair if anyone tries to actually store `null` in a form data element. They also implicitly check for null by checking the Error attribute, although I'm not sure we can fix that with annotations.

It is also an old version of app-lib:
https://altinn.studio/repos/brg/starteogdrive/src/commit/e3b62d445a112e2db65004b90c1f0575ee3c52ab/App/App.csproj#L12

> `<PackageReference Include="Altinn.App.Api" Version="7.13.1">`

Reply:
Ahh ok nice